pheat.F90

```diff
@@ -211,6 +211,13 @@ subroutine pheat(carma, cstate, iz, igroup, iepart, ibin, igas, dmdt, rc)
                     * rhoelem(ibin,iepart) / 2._f / dtime, &
                    dmdt)
       end if
+    elseif(neutral_volfrc(igroup) .eq. -1._f)then
+      ! When the particle neutralization is not controled by the core mass, 
+      ! instead it is controled by ambience gas
+      ! for example, in trop_strat model, sulfate in mixed particle is neutralized
+      ! by NH4. In this case, we set the neutral_volfrc = -1 for the group
+      dmdt = max((pvap * (ss + 1._f)) * g0, dmdt)
+      !write(*,*) "igroup",igroup," in pheat.F90:neutral_volfrc"
```

Comment:
Need to think about this a little. I think What this is doing is saying that gas can't leave the particle, but it can go on the particle. The comment here isn't very clear to me about what this is trying to do. How is the ambient gas controlling the evaporation? This just seems like it is shutting it off.

Reply:
Might put this change in a separate pull request?

Reply:
Yes,  lets leave this out here and do a new pull request afterwards

Reply:
@zywshoon  change comment: The sulfate does not evaporate in the mixed particles, because it is assumed that it is neutralized by ammonia.
(@drmikemills need to check if this is a problem in the WACCM model, test this without the change, Yunqian is providing Mike the change to test just applying this in the lower stratosphere.
(@fvitt  keep in the sandbox)

Reply:
done